name.abuchen.portfolio/src/name/abuchen/portfolio/model/Transaction.java

```diff
@@ -144,6 +144,7 @@ public int compare(Transaction t1, Transaction t2)
 
     private LocalDateTime date;
     private String currencyCode;
+    private boolean isDynamicCurrencyCode;
```

Comment:
Adding a property to any class in the "model" packages has an immediate impact on the XML model. It means the property will show up in the XML and we cannot remove it anymore.

One alternative is that we define the property as "transient". Then the field is not stored and read from the XML file.

However, in this case I propose to check the property value itself. There are two options:

* we can leave the transaction ```null``` which identifies that no transaction has been found. I believe that would pretty much reflect the result of the import: no currency found. However, as PP is also often working with ```Money``` and that class requires a currency,
* we can use a special value - I would prefer ```UNDEFINED``` over ```DYNAMIC```. Instead of checking with an additional property, we can check with ```"UNDEFINED".equals(tx.getCurrencyCode())```